### DISCUSSION TITLE
Pagination blocks bootstrap downdown menu in table row when expanded

### DISCUSSION THREAD

**Discussion by geradt** [primefaces/community#125] [PrimeVue]
I am using Bootstrap on the front end for styling and some components. One situation I'm running into is that I have a bootstrap downdown menu button that was placed into the datatable row at the end as an action button (Move to). And in the dropdown menu is a list of places to move the record to. It's working great, the only issue I have is when I'm trying to view the dropdown menu in the lower rows of the table. When the dropdown menu button expands the list of options the options are cut off by the end of the table and start of pagination (it goes behind). IS there any way to easily make it so the dropdown menu doesn't get cut off by the pagination? I tried playing with z-index's and I wasn't able to get the desired effect. Any thoughts? Thanks!

**Comment by tugcekucukoglu** [comment]
As I understand you are using DataTable and Paginator with PrimeVue Bootstrap theme. And there is a Button component that shows a menu when it's clicked in the last column of this DataTable. Can you send us a reproducer link where we can test this properly?
